docs/source/intro.rst

```diff
@@ -0,0 +1,151 @@
+   https://dcc.icgc.org/icgc-in-the-cloud/aws
+-  HCA Storage: see https://dss.staging.data.humancellatlas.org/ and
+   https://github.com/HumanCellAtlas/data-store
+-  the GDC Storage: see https://gdc.cancer.gov
+-  Keep by Curoverse: see https://arvados.org/ and
+   https://github.com/curoverse/arvados
+
+The goal of DOS is to create a generic API on top of these and other
+projects, so workflow systems can access data in the same way regardless
+of project. One section of the API focuses on how to read and write data
+objects to cloud environments and how to join them together as data
+bundles (Data object management). Another focuses on the ability to find
+data objects across cloud environments and implementations of DOS (Data
+object queries). The latter is likely to be worked on in conjunction
+with the GA4GH Discovery Workstream.
+
+Key features of the current API proposal:
+
+Data object management
+^^^^^^^^^^^^^^^^^^^^^^
+
+This section of the API focuses on how to read and write data objects to
+cloud environments and how to join them together as data bundles. Data
+bundles are simply a flat collection of one or more files. This section
+of the API enables:
+
+-  create/update/delete a file
+-  create/update/delete a data bundle
+-  register UUIDs with these entities (an optionally track versions of
+   each)
+-  generate signed URLs and/or cloud specific object storage paths and
+   temporary credentials
+
+Data object queries
+^^^^^^^^^^^^^^^^^^^
+
+A key feature of this API beyond creating/modifying/deletion files is
+the ability to find data objects across cloud environments and
+implementations of DOS. This section of the API allows users to query by
+data bundle or file UUIDs which returns information about where these
+data objects are available. This response will typically be used to find
+the same file or data bundle located across multiple cloud environments.
+
+Implementations
+---------------
+
+There are currently a few experimental implementations that use some
+version of these schemas.
+
+-  `DOS Connect <https://github.com/ohsu-comp-bio/dos_connect>`__
+   observes cloud and local storage systems and broadcasts their changes
+   to a service that presents DOS endpoints.
+-  `DOS Downloader <https://github.com/david4096/dos-downloader>`__ is a
+   mechanism for downloading Data Objects from DOS URLs.
+-  `dos-gdc-lambda <https://github.com/david4096/dos-gdc-lambda>`__
+   presents data from the GDC public rest API using the Data Object
+   Service.
+-  `dos-signpost-lambda <https://github.com/david4096/dos-signpost-lambda>`__
+   presents data from a signpost instance using the Data Object Service.
+
+Building the client and server
+------------------------------
+
+You can use ``pip`` to install a python client and server that
+implements these schemas.
+
+::
+
+    virtualenv env
+    source env/bin/activate
+    pip install git+git://github.com/ga4gh/data-object-service-schemas@master --process-dependency-links
```

Comment:
Is there a reason this isn't `pip install ga4gh-dos-schemas `?

Reply:
Newp, just hadn't published when this was made I believe. +1 to deleting/moving a lot of cruft on this readme and replacing with "pip install" and a client demo.